workbench-core/authentication/.eslintrc.js

```diff
@@ -3,5 +3,9 @@ require('@rushstack/eslint-config/patch/modern-module-resolution');
 
 module.exports = {
   extends: ['@aws/eslint-config-workbench-core-eslint-custom'],
-  parserOptions: { tsconfigRootDir: __dirname }
+  parserOptions: { tsconfigRootDir: __dirname },
+  // These rules are turned off to be compatible with aws-jwt-verify exported components
+  rules: {
+    'import/namespace': ['off']
+  }
```

Comment:
would it make sense to turn this rule off just for where its necessary? or is it flagged in a lot of places?